src/architecture/msgPayloadDefC/TypesTestMsgPayload.h

```diff
@@ -0,0 +1,66 @@
+/*
+ ISC License
+
+ Copyright (c) 2024, Astroscale Japan
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#ifndef TYPES_TEST_MESSAGE_H
+#define TYPES_TEST_MESSAGE_H
+
+#include <stdint.h>
+
+#define TYPES_TEST_ARRAY_SIZE 3
+
+/*! @brief Structure used to test the handling of various built-in types in message payloads across SWIG C/Python bridge */
+typedef struct {
+    // Scalars
+    int8_t i8Test;      //!< [] Test variable of int8_t
```

Comment:
@schaubh I added comments here. Are there any other places that comments need to be added?

Reply:
I pulled your fork and tested the branch.  The doxygen build warnings are now gone.  Thanks for doing that.  

For these variables there are no units.  I'm not using `[]` anymore as this is not needed in these cases, and can create odd HTML issues at times.

Could you please 
- remove the `[]` characters in that commit, squash to a single commit again.
- rebase on latest develop

Reply:
Done!